### PR TITLE
User manager

### PR DESCRIPTION
**Pull request by jordic**

### CODE REVIEW

**Comment by jordic** [comment]
It still misses the new context var parts..

File: guillotina_dbusers/content/groups.py [hunk 1]
```diff
@@ -53,6 +54,7 @@ def properties(self):
     type_name="GroupManager",
     schema=IGroupManager,
     behaviors=["guillotina.behaviors.dublincore.IDublinCore"],
-    allowed_types=["Group"])
+    allowed_types=["Group"],
```
Comment (vangheem) [inline guillotina_dbusers/content/groups.py:53]: groups within groups? Do we want to support?

**Review by vangheem — APPROVED** [review]
lgtm

File: guillotina_dbusers/content/groups.py (hunk 1 above)
Comment (jordic) [reply guillotina_dbusers/content/groups.py:53]: absolutely no! haha

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-2.0.4.dev0
+2.1.0.dev0
```
Comment (bloodbare) [inline VERSION]: 5.0.0

**Review by bloodbare — APPROVED** [review]

**Pull request merged**